File: rss-reader/app/src/main/java/com/abubusoft/kripton/examples/rssreader/service/api/BbcApi.java
package com.abubusoft.kripton.examples.rssreader.service.api;

import com.abubusoft.kripton.examples.rssreader.service.model.RssFeed;

import retrofit2.Call;
import retrofit2.http.GET;

public interface BbcApi {
    @GET("news/rss.xml?edition=int")
    Call<RssFeed> loadRSSFeed();
}

File: kotlin-rss-reader/app/src/main/java/com/abubusoft/rssreader/service/api/BbcApi.kt
package com.abubusoft.rssreader.service.api

import com.abubusoft.rssreader.service.model.RssFeed
import retrofit2.Call
import retrofit2.http.GET

interface BbcApi {
    @GET("news/rss.xml?edition=int")
    fun loadRSSFeed(): Call<RssFeed>
}